### ISSUE TITLE
Not requesting an adequately resized artwork

### ISSUE THREAD

**Issue by kayue** [CDrummond/lms-material#100]
I'm using the Qobuz plugin, and I noticed the image returning from image proxy is larger than the original one from Qobuz server.

Image proxy returning 1500px 

```
/imageproxy/https%3A%2F%2Fstatic.qobuz.com%2Fimages%2Fcovers%2F52%2F13%2F0094636981352_600.jpg/image.jpg
```

Original picture is 600px

```
https://static.qobuz.com/images/covers/52/13/0094636981352_600.jpg
```

Expected result would be something like this

```
/imageproxy/https%3A%2F%2Fstatic.qobuz.com%2Fimages%2Fcovers%2F52%2F13%2F0094636981352_600.jpg/image_300x300_f.png
```

The bug was originally reported at https://github.com/Logitech/slimserver/issues/247, then they lead me to here.

**Comment by CDrummond** [comment]
And this is an issue because? Material uses the unscaled images for now-playing and backdrops. Why should these be scaled? Perhaps the cover in the bottom toolbar of the desktop view should be scaled. But I fail to see why it should elsewhere.

**Comment by CDrummond** [comment]
http://192.168.1.16:9000/imageproxy/https%3A%2F%2Fstatic.qobuz.com%2Fimages%2Fcovers%2F52%2F13%2F0094636981352_600.jpg/image.jpg

is 300x300 for me

**Comment by CDrummond** [comment]
Resolved by #101
